libraries/adaptive-expressions/babel.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (api) {
+    api.cache(false);
+    const presets = [
+        ['@babel/preset-typescript'],
+        [
+            '@babel/preset-env',
+            {
+                corejs: { version: 3 },
+                useBuiltIns: 'usage',
+                targets: {
+                    browsers: ['last 2 versions', 'ie >= 11', 'not ie < 11', 'not android > 0', 'not ios > 0'],
+                },
+            },
+        ],
+    ];
+    const plugins = [
+        ['@babel/plugin-proposal-decorators', { decoratorsBeforeExport: true }],
+        ['@babel/plugin-proposal-class-properties'],
+        ['@babel/plugin-transform-modules-commonjs'],
+        ['@babel/plugin-transform-runtime', { useESModules: false }],
+    ];
```

Comment:
This is toolchain config and not consumed by end-users, but it would be great if you added some comments explaining what's going on here for future maintainers who may not be familiar with the current scenario and/or babel ecosystem.

Reply:
Added comments . Let me know if its okay